src/display/editor/editor.js

```diff
@@ -1016,8 +1031,24 @@ class AnnotationEditor {
     this.#altText.data = data;
   }
 
+  get guessedAltText() {
+    return this.#altText.guessedText;
+  }
+
+  async setGuessedAltText(text) {
+    await this.#altText.setGuessedText(text);
+  }
+
+  serializeAltText(isForCopying) {
+    return this.#altText?.serialize(isForCopying);
+  }
+
   hasAltText() {
-    return !this.#altText?.isEmpty();
+    return this.#altText && !this.#altText.isEmpty();
```

Comment:
If `this.#altText`  is `null` this function returns `null` while I'd expect this to always return a boolean. Should we use `!!this.altText` instead?